corehq/apps/api/user_updates.py

```diff
@@ -165,3 +167,53 @@ def _log_phone_number_change(new_phone_numbers, old_phone_numbers, user_change_l
 
     if change_messages:
         user_change_logger.add_change_message({'phone_numbers': change_messages})
+
+
+def _update_location(user, location_object, user_change_logger):
```

Comment:
What would you think about pulling parts of this out into a couple helper functions?

Reply:
refactored in ca474379fe01a01e967cdc77dbf5d8d24ad1c64d